src/victory-util/prop-types.js

```diff
@@ -127,11 +123,12 @@ export const domain = makeChainable((props, propName, componentName) => {
  */
 export const scale = makeChainable((props, propName, componentName) => {
   const supportedScaleStrings = ["linear", "time", "log", "sqrt"];
+  const isFunction = (val) => typeof val === "function";
```

Comment:
There are [good reasons](https://github.com/lodash/lodash/blob/npm-packages/lodash.isfunction/index.js#L40-L42) to `_.isFunction` here, for a minor hit in bundle size.